### PR TITLE
Renamed consensusStartupErrorTracer

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/Tracers.hs [hunk 1]
```diff
@@ -81,7 +81,7 @@ instance (forall a. Semigroup (f a))
       , blockchainTimeTracer          = f blockchainTimeTracer
       , forgeStateInfoTracer          = f forgeStateInfoTracer
       , keepAliveClientTracer         = f keepAliveClientTracer
-      , consensusStartupErrorTracer   = f consensusStartupErrorTracer
+      , consensusErrorTracer          = f consensuErrorTracer
```
Comment (jasagredo) [inline ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/Tracers.hs:84]: I think there is a typo here: "consensu"

**Review by jasagredo — COMMENTED** [review]
I'm still confused, is this the one that says "There has been an exception but the exception is ExitSuccess"? If so then maybe this shouldn't be called "error tracer"?

**Comment by coot** [comment]
Are you throwing `ExitSuccess` in the consensus code (e.g. using `exitSuccess`)?

File: ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/Tracers.hs (hunk 1 above)
Comment (coot) [reply ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/Tracers.hs]: Fixed.

**Review by jasagredo — APPROVED** [review]
Looks good thanks!

**Comment by coot** [comment]
@jasagredo I found a nicer way to combine predicates of `handleJust`, by using the `Predicate` monoid.

I also fixed `script/ci/check-changelogs.sh`.

**Review by jasagredo — APPROVED** [review]

**Comment by coot** [comment]
`GHA` windows [job](https://github.com/input-output-hk/ouroboros-consensus/actions/runs/4955346620/jobs/8864671530?pr=71) failed with:
```
C://ghcup//ghc//8.10.7//mingw//bin/as.exe: out of memory allocating 8 bytes
`gcc.exe' failed in phase `Assembler'. (Exit code: 1)
```

I will restart and :crossed_fingers:.

**Pull request merged**